src/runtime_src/core/common/message.cpp

```diff
@@ -62,18 +62,6 @@ get_userid()
 #endif
 }
 
-static std::string
-get_hostname()
```

Comment:
Since I moved the get_hostname function into core/common/unistd.h I figured fixing this would also be a good idea.